bakage.pl

```diff
@@ -14,6 +26,36 @@
 :- use_module(library(iso_ext)).
 :- use_module(library(debug)).
 
+main :- 
```

Comment:
I have made good experiences with calling such a predicate **`run/0`**, maybe this is worth considering.